<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentDownload</class>
 <widget class="QWidget" name="ContentDownload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="category"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="currentCategoryLabel">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentCategoryIdLabel">
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="search"/>
   </item>
   <item>
    <widget class="QListWidget" name="contentList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
